Formula/geesefs.rb

```diff
@@ -0,0 +1,24 @@
+class Geesefs < Formula
+  desc "Finally, a good FUSE FS implementation over S3"
+  homepage "https://github.com/yandex-cloud/geesefs"
+  url "https://github.com/yandex-cloud/geesefs/archive/refs/tags/v0.30.5.tar.gz"
+  sha256 "149ca98073641b8c051a361950a7111520be1da032489614d8fdcb1316543bc6"
+
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  # For tests
+  depends_on "bash" => :build
+  depends_on "python@3.10" => :build
+  depends_on "truncate" => :build
+  depends_on "openjdk" => :build
+
+  def install
+    system "go", "build", "-o", "#{bin}/geesefs"
+  end
+
+  test do
+    system "bash", "test/run-tests.sh"
```

Comment:
Is this something a user would commonly run?

Reply:
No, definitely not